.vscode/settings.json

```diff
@@ -72,5 +70,9 @@
   "eslint.options": {
     "configFile": "./.eslintrc.json"
   },
-  "eslint.validate": ["typescript", "typescriptreact"]
+"eslint.validate": [
+    "typescript",
+    "typescriptreact"
+],
+"python.pythonPath": "/Users/suntao/anaconda3/envs/py38/bin/python"
```

Comment:
@suniique make sure not to add your personal information in the commit.

Reply:
OK, fixed.